cluster/selfplay/tpu_wrapper.sh

```diff
@@ -31,8 +31,8 @@ bazel-bin/cc/main \
   --games_per_inference=16 \
   --parallel_games=32 \
   --num_readouts=800 \
```

Comment:
did we discuss moving this to 1200?

Reply:
we did, but i'm not sure yet that we'll have the extra quota for v10 (and not v11)